src/net/p2p.rs

```diff
@@ -2165,8 +2178,19 @@ impl PeerNetwork {
                     return Ok(None);
                 }
             }
-            (false, _) => {
-                return Ok(None);
+            (false, txs_opt) => {
+                // did we get some transactions, but have more to get?
+                if let Some(txs) = txs_opt {
+                    debug!(
+                        "{:?}: Mempool sync obtained {} transactions from mempool sync, but have more",
+                        &self.local_peer,
+                        txs.len()
+                    );
+
+                    return Ok(Some(txs));
```

Comment:
The `if let Some(...)` branch here appears to not be covered

Reply:
I don't think that code is even reachable.  I will just remove it instead.

Reply:
Hmm, it's definitely getting exercised.  I wonder why codecov didn't report it.

Reply:
Yes -- that'd be why -- I think the `#[ignore]` tests aren't getting executed by the code coverage unit tests.